permission_handler_android/lib/src/android_object_mirrors/activity.dart

```diff
@@ -116,17 +116,17 @@ class Activity extends JavaObject {
   ///
   /// See https://developer.android.com/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int).
   Future<ActivityResult> startActivityForResult(
-    Intent intent,
+    Intent intent, [
     int? requestCode,
-  ) {
+  ]) {
```

Comment:
Can we make the `requestCode` an optional named parameter, instead of an optional positional parameter?

Named parameters are more developer friendly as they show up better in the IDE and also allows for a bit more flexibility in case we need to add additional parameters.